components/buttons.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from "react";
+
+interface Props {
+  text: string
+  type: string
+  size: string
+}
+
+export const Buttons: FC<Props> = ({ text, type, size }) => {
+  let whichButton: string = type + size;
```

Comment:
You can use `const` and name it `styleType` because we don't need to specify button because we are in the Button class.

Reply:
we cannot use const, because we are changing the value of the variable depends on the button style we want to use